Editor/BacktraceConfigurationLabels.cs

```diff
@@ -10,6 +10,13 @@ internal static class BacktraceConfigurationLabels
         internal static string LABEL_DESTROY_CLIENT_ON_SCENE_LOAD = "Destroy client on new scene load (false - Backtrace managed)";
         internal static string LABEL_SAMPLING = "Log random sampling rate";
         internal static string LABEL_HANDLE_ANR = "Handle ANR (Application not responding)";
+
+        internal static string LABEL_HANDLE_OOM =
+#if UNITY_ANDROID
+            "(Early access) Send Low memory warnings to Backtrace";
+#else
+            "(Early access) Send Out of memory exceptions to Backtrace";
```

Comment:
Maybe we can guard this `else` statement a bit more, unless all other platforms support out of memory exceptions

Reply:
done.